### ISSUE TITLE
`enabled = false` doesn't disable a plugin

### ISSUE THREAD

**Issue by t-botz** [terraform-linters/tflint#1397]
The following config doesn't disable aws
```hcl
config {
  module = false
  force = false
}

plugin "aws" {
  enabled = false
}
```

On the other, this does disable it:

```hcl
config {
  module = false
  force = false
}
```

I don't think there is much usecase for `enabled = false` so maybe the best fix is to remove the `enabled` flag altogether and just allow
 
```hcl
plugin "aws" { }
```

## Version

0.36.2

**Comment by wata727** [comment]
As far as I've tried, `enabled = false` works fine. Is there a minimum reproduction configuration?
